etc/osg-koji-tags/osg-tags.exclude

```diff
@@ -1,3 +1,13 @@
+osg-3.3-el6-contrib
+osg-3.3-el6-empty
+osg-3.3-el6-development
+osg-3.3-el6-testing
+osg-3.3-el6-release
+osg-3.3-el7-contrib
+osg-3.3-el7-empty
+osg-3.3-el7-development
+osg-3.3-el7-testing
+osg-3.3-el7-release
 osg-3.2-el7-release
 osg-3.1-el5-contrib
 osg-3.1-el5-development
```

Comment:
I guess we can drop these 3.1/3.2 tags now also since the tags have been removed from koji

```suggestion
```

Reply:
As a note - probably i should allow globs in the exclude file to let you just put a `osg-3.3-*` in there.  But i don't think that's supported currently.